src/clr-core/button/button.element.ts

```diff
@@ -5,18 +5,216 @@
+ * @cssProp `--clr-btn-success-disabled-bg-color`
+ * @cssProp `--clr-btn-success-disabled-border-color`
+ * @cssProp `--clr-btn-success-checked-bg-color`
+ * @cssProp `--clr-btn-success-checked-color`
+ * @cssProp `--clr-btn-success-outline-color`
+ * @cssProp `--clr-btn-success-outline-bg-color`
+ * @cssProp `--clr-btn-success-outline-border-color`
+ * @cssProp `--clr-btn-success-outline-hover-bg-color`
+ * @cssProp `--clr-btn-success-outline-hover-color`
+ * @cssProp `--clr-btn-success-outline-box-shadow-color`
+ * @cssProp `--clr-btn-success-outline-disabled-color`
+ * @cssProp `--clr-btn-success-outline-disabled-bg-color`
+ * @cssProp `--clr-btn-success-outline-disabled-border-color`
+ * @cssProp `--clr-btn-success-outline-checked-bg-color`
+ * @cssProp `--clr-btn-success-outline-checked-color`
+ * @cssProp `--clr-btn-danger-color`
+ * @cssProp `--clr-btn-danger-bg-color`
+ * @cssProp `--clr-btn-danger-border-color`
+ * @cssProp `--clr-btn-danger-hover-bg-color`
+ * @cssProp `--clr-btn-danger-hover-color`
+ * @cssProp `--clr-btn-danger-box-shadow-color`
+ * @cssProp `--clr-btn-danger-disabled-color`
+ * @cssProp `--clr-btn-danger-disabled-bg-color`
+ * @cssProp `--clr-btn-danger-disabled-border-color`
+ * @cssProp `--clr-btn-danger-checked-bg-color`
+ * @cssProp `--clr-btn-danger-checked-color`
+ * @cssProp `--clr-btn-danger-outline-color`
+ * @cssProp `--clr-btn-danger-outline-bg-color`
+ * @cssProp `--clr-btn-danger-outline-border-color`
+ * @cssProp `--clr-btn-danger-outline-hover-bg-color`
+ * @cssProp `--clr-btn-danger-outline-hover-color`
+ * @cssProp `--clr-btn-danger-outline-box-shadow-color`
+ * @cssProp `--clr-btn-danger-outline-disabled-color`
+ * @cssProp `--clr-btn-danger-outline-disabled-bg-color`
+ * @cssProp `--clr-btn-danger-outline-disabled-border-color`
+ * @cssProp `--clr-btn-danger-outline-checked-bg-color`
+ * @cssProp `--clr-btn-danger-outline-checked-color`
+ * @cssProp `--clr-btn-link-color`
+ * @cssProp `--clr-btn-link-bg-color`
+ * @cssProp `--clr-btn-link-border-color`
+ * @cssProp `--clr-btn-link-hover-bg-color`
+ * @cssProp `--clr-btn-link-hover-color`
+ * @cssProp `--clr-btn-link-disabled-color`
+ * @cssProp `--clr-btn-link-disabled-bg-color`
+ * @cssProp `--clr-btn-link-disabled-border-color`
+ * @cssProp `--clr-btn-link-checked-bg-color`
+ * @cssProp `--clr-btn-link-checked-color`
+ * @cssProp `--clr-btn-inverse-color`
+ * @cssProp `--clr-btn-inverse-border-color`
+ * @cssProp `--clr-btn-inverse-bg-color`
+ * @cssProp `--clr-btn-inverse-hover-bg-color`
+ * @cssProp `--clr-btn-inverse-hover-color`
+ * @cssProp `--clr-btn-inverse-box-shadow-color`
+ * @cssProp `--clr-btn-inverse-disabled-color`
+ * @cssProp `--clr-btn-inverse-disabled-bg-color`
+ * @cssProp `--clr-btn-inverse-disabled-border-color`
+ * @cssProp `--clr-btn-inverse-checked-bg-color`
+ * @cssProp `--clr-btn-inverse-checked-color`
+ * @cssProp `--clr-btn-icon-disabled-color`
+ * @cssProp `--clr-btn-group-focus-outline`
+ */
```

Comment:
Wow. I went _EXTRA_ on the CSS props for buttons. Sorry about that. 😰

Reply:
I wish there was a way for the TSDoc/API docs to statically check what properties there are without comments. But for now I think this is the best we have